AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -68,12 +69,19 @@ - (void)_staticInitialize;
 
 @implementation ASDisplayNode
 
-// these dynamic properties all defined in ASLayoutOptionsPrivate.m
-@dynamic spacingAfter, spacingBefore, flexGrow, flexShrink, flexBasis,
-         alignSelf, ascender, descender, sizeRange, layoutPosition, layoutableType;
+// Dynamic properties for ASLayoutables
+@dynamic layoutableType, size;
+
+// Dynamic properties for sizing
+@dynamic width, height, minWidth, maxWidth, minHeight, maxHeight;
+@dynamic widthAsPoints, widthAsFraction, minWidthAsPoints, minWidthAsFraction, maxWidthAsPoints, maxWidthAsFraction;
```

Comment:
I'm not sure how I feel about all these as opposed to having functions for setting sizes in percentages or points. Also, what will something like widthAsPoints return after you've set it with widthAsFraction?